src/Data/Vec/Properties.agda

```diff
@@ -364,6 +364,10 @@ map-id : ∀ {n} → map {A = A} {n = n} id ≗ id
 map-id []       = refl
 map-id (x ∷ xs) = P.cong (x ∷_) (map-id xs)
 
+map-const : ∀ {n} (xs : Vec A n) (x : B) → map {n = n} (const x) xs ≡ replicate x
```

Comment:
Very minor comment, but could we rename the constant value from `x`, e.g. `v`. Makes it clearer that it's not an element of the list `xs`...